.github/workflows/e2e-release.yml

```diff
@@ -53,13 +53,10 @@ jobs:
         set -x
         docker pull ${{ steps.website_variables.outputs.cloudshell_image }}
         # run install script
-        # ISTIO_VERSION is fix for v0.5.0. Can be removed after next release
         docker run --rm \
           -e project_id=${{ env.PROJECT_ID }} \
-          -e skip_workspace_prompt=1 \
           -e release_repo=${{ steps.website_variables.outputs.repo }} \
           -e release_branch=${{ steps.website_variables.outputs.branch }} \
-          -e ISTIO_VERSION=1.7.1 \
```

Comment:
how is this related to your PR?

Reply:
It's semi-relevant since it's another abandoned flag I found as I was working. But we don't have to address this here